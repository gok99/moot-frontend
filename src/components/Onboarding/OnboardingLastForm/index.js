import React from 'react';
import { Button, Row, Col, ProgressBar } from 'react-bootstrap';

import '../../Styles/styles.css';
import '../onboarding.css';
import logo from '../../../assets/navlogo.png';

/**
 * Functional Presentational Component that displays Onboarding Form 7.
 */
const OnboardingLastForm = (props) => {
  const onSubmit = props.onSubmit;

  return (
    <Col>
      <Row className="d-flex justify-content-center">
        <Col md="auto">
          <img className="logo-onboarding" src={logo} alt="moot" />
        </Col>
      </Row>
      <Row>
        <p><br /></p>
        <p className="header-onboarding">Your account's ready to go!</p>
        <p><br /></p>
      </Row>
      <Row>
        <Button className="btn-onboarding" type="button" onClick={onSubmit}>Start</Button>
      </Row>
      <Row>
        <p><br /><br /></p>
        <ProgressBar striped variant="info" animated now={100} />
        <p><br /></p>
      </Row>
    </Col>
  );
};

export default OnboardingLastForm;