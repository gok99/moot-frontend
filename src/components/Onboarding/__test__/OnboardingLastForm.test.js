import React from 'react';
import ReactDOM from 'react-dom';
import OnboardingLastForm from '../OnboardingLastForm';

import { cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect"
import renderer from 'react-test-renderer';

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement('div');
  ReactDOM.render(<OnboardingLastForm onSubmit={(event) => { event.preventDefault(); }} />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("matches snapshot", () => {
  const tree = renderer.create(<OnboardingLastForm onSubmit={(event) => { event.preventDefault(); }} />).toJSON();
  expect(tree).toMatchSnapshot();
});