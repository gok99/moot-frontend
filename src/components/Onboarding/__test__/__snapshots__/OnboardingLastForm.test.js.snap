// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="col"
>
  <div
    className="d-flex justify-content-center row"
  >
    <div
      className="col-md-auto"
    >
      <img
        alt="moot"
        className="logo-onboarding"
        src="navlogo.png"
      />
    </div>
  </div>
  <div
    className="row"
  >
    <p>
      <br />
    </p>
    <p
      className="header-onboarding"
    >
      Your account's ready to go!
    </p>
    <p>
      <br />
    </p>
  </div>
  <div
    className="row"
  >
    <button
      className="btn-onboarding btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Start
    </button>
  </div>
  <div
    className="row"
  >
    <p>
      <br />
      <br />
    </p>
    <div
      className="progress"
    >
      <div
        aria-valuemax={100}
        aria-valuemin={0}
        aria-valuenow={100}
        className="progress-bar bg-info progress-bar-animated progress-bar-striped"
        role="progressbar"
        style={
          Object {
            "width": "100%",
          }
        }
      />
    </div>
    <p>
      <br />
    </p>
  </div>
</div>
`;
